src/platform_impl/windows/event_loop.rs

```diff
@@ -1448,8 +1448,17 @@ unsafe extern "system" fn public_window_callback<T>(
             {
                 let dpi_factor = hwnd_scale_factor(window);
                 for input in &inputs {
-                    let x = (input.x as f64) / 100f64;
-                    let y = (input.y as f64) / 100f64;
+                    let mut location = POINT {
+                        x: input.x / 100,
+                        y: input.y / 100,
+                    };
```

Comment:
Doing the conversion like this loses the subpixel precision that the windows API provides. It should be possible to do the conversion to client-space and bring over the remainder.